GW2EIBuilders/Resources/htmlHealingExtensionTemplates/DistributionTables/tmplBarrierDistPlayer.html

```diff
@@ -14,10 +14,12 @@
         <div class="d-flex flex-row justify-content-center mt-1 mb-1">
             <ul class="nav nav-pills scale85">
                 <li class="nav-item">
-                    <a class="nav-link" @click="targetmode = 1" :class="{active: targetmode}">Target</a>
+                    <a class="nav-link" @click="targetmode = 1" data-original-title="Against selected target(s)."
+                       :class="{active: targetmode}">Target</a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" @click="targetmode = 0" :class="{active: !targetmode }">All</a>
+                    <a class="nav-link" @click="targetmode = 0" data-original-title="Against all targets."
```

Comment:
You should change "all targets" with "all enemies". Target word could be confused with the concept of selectable targets.